webviz_subsurface/_utils/colors.py

```diff
@@ -63,29 +55,50 @@ def hex_to_rgba_str(hex_string: str, opacity: float = 1.0) -> str:
 
 def rgb_to_str(rgb: Tuple[float, float, float]) -> str:
     """Convert rgb tuple with floating point byte color values to string
-
     `Input:`
     * rgb - Tuple[float,float,float] - RGB color on tuple format, r-, g- and b-channel
     is index 0, 1 and 2, respectively. Byte color values 0-255
-
     `Return:`
     RGB color on string format "rgb(r,g,b)" where, channels r, g and b are
     represented with byte color integer value 0-255.
     """
     return f"rgb({round(rgb[0])}, {round(rgb[1])}, {round(rgb[2])})"
 
 
+def rgba_to_str(rgba: Tuple[float, float, float, float]) -> str:
+    """Convert rgba tuple with floating point byte color values to string
+    `Input:`
+    * rgb - Tuple[float,float,float,float] - RGBA color on tuple format
+    `Return:`
+    RGBA color on string format "rgba(r,g,b,a)" where, channels r, g and b are
+    represented with byte color integer value 0-255, and a is represented by a
+    decimal number 0-1"""
+    return f"rgba({round(rgba[0])}, {round(rgba[1])}, {round(rgba[2])}, {rgba[3]})"
```

Comment:
Feels a bit strange to have both `rgba_to_str` and `rgb_to_str`, but there are also other (existing) functions in `_utils/colors` which might be condensed perhaps. We leave it as is, but maybe we should make an issue on reducing number of color-functions if natural.